onnxruntime/core/framework/onnxruntime_typeinfo.cc

```diff
@@ -37,6 +61,28 @@ ORT_API_STATUS_IMPL(OrtApis::CastTypeInfoToTensorInfo, _In_ const struct OrtType
   return nullptr;
 }
 
+ORT_API_STATUS_IMPL(winmla::CastTypeInfoToMapTypeInfo, const OrtTypeInfo* type_info, const OrtMapTypeInfo** out) {
```

Comment:
>winmla [](start = 20, length = 6)

These adapter apis are implemented in onnxruntime_typeinfo, they can technically move into the adapter folder without issue, but i think they really belong here and in the c-abi...

Reply:
moved implementations into onnxruntime/core/framework

---
In reply to: [370287686](https://github.com/microsoft/onnxruntime/pull/2891#discussion_r370287686) [](ancestors = 370287686)